src/MyBrokenClass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class MyBrokenClass
+{
+    protected string $test;
+
+    public function test(): string
+    {
+        foreach ( $_POST['menu-locations'] as $menu_id ) {
```

Comment:
:no_entry_sign: **Error**: Detected usage of a possibly undefined superglobal array index: $_POST['menu-locations']. Use isset() or empty() to check the index exists before using it (*WordPress.Security.ValidatedSanitizedInput.InputNotValidated*).